conda.recipe/lux-api/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "lux-api" %}
+{% set version = "0.2.2" %}
```

Comment:
Is it possible to read this number directly from [_version.py](https://github.com/lux-org/lux/blob/master/lux/_version.py) so that we wouldn't have to update the recipe every time there is a new release? A similar thing that reads from Github tags is done [here](https://github.com/modin-project/modin/blob/master/scripts/conda-recipe/meta.yaml)

Reply:
I can try this approach out, I have already uploaded 0.2.2 to conda-forge so I will try it for the new release

Reply:
Ok that sounds good!